spec/Schema.spec.js

```diff
@@ -1734,4 +1734,29 @@ describe('Class Level Permissions for requiredAuth', () => {
         }
       );
   });
+
+  it('should not throw on null field types', async () => {
+    const schema = await config.database.loadSchema();
+    const result = await schema.enforceFieldExists(
+      'NewClass',
+      'fieldName',
+      null
+    );
+    expect(result).toBeUndefined();
+  });
+
+  it('ensureFields should throw when schema is not set', async () => {
```

Comment:
same thing here. I'd use the "await expectAsync(...).toBeRejectedWith(...);" format so we can ensure it is really getting into the catch block.